src/config.js

```diff
@@ -4,4 +4,6 @@ export default {
   compatibilityUrl: 'https://zulipchat.com/compatibility',
   defaultLoginEmail: '', // useful during development, empty for production
   defaultLoginPassword: '', // useful during development, empty for production
+  appStoreLink: 'https://itunes.apple.com/us/app/facebook/id284882215?mt=8',
```

Comment:
Yeah, those are placeholders for testing purposes - will be sure to remove them after final review :)